cli/pcluster/config/hit_converter.py

```diff
@@ -73,6 +78,14 @@ def convert(self):
                     sit_cluster_section.get_param("placement_group"), queue_section.get_param("placement_group")
                 )
 
+                # Print a warning for unsupported parameters
+                if sit_cluster_section.get_param_value("placement") == "cluster":
+                    LOGGER.log(
+                        log_level,
+                        "Warning: 'placement = cluster' is not supported in the Heterogeneous Instance Type model "
```

Comment:
How are we referring to this elsewhere? Are we calling it Heterogeneous Instance types everywhere else? If so, ignore this comment suggesting we refer to it as multiple instance types (as well as other similar comments I've left).

"Warning: 'placement = cluster' is not supported when using multiple instance types."

Reply:
Totally agree with the comment, HIT was a jargon we use it's not defined anywhere, let's use multiple instance types configurations, and or multiple queues multiple instance types.. something that make the things more clear to a broad audience. Also not capitalized, it's not the proper noun of a component / technology

Reply:
Fixed, thanks for the feedback.

Reply:
"multiple queues" makes sense to me; that aligns well with the configuration settings. If we were to use "multiple instance types" I'd suggest something like "multiple compute instance types" since the head node could already be a different instance type.

Reply:
@HLeeHart: You raise a good point. I think we should refer to _both_ multiple queues and multiple instance types as separate features (because they can be used independently of each other). And you're right that we want to be careful in clarifying that we mean multiple _compute_ instance types, although I think it's reasonable for succinctness to shorten that in some places to just multiple instance types (which is generally how customers talk about this feature when they have requested it in the past).